vector/src/main/java/im/vector/riotx/features/roomprofile/RoomProfileViewModel.kt

```diff
@@ -111,11 +113,13 @@ class RoomProfileViewModel @AssistedInject constructor(@Assisted private val ini
     private fun handleChangeAvatar(action: RoomProfileAction.ChangeRoomAvatar) {
         _viewEvents.post(RoomProfileViewEvents.Loading())
         room.rx().updateAvatar(action.uri, action.fileName ?: UUID.randomUUID().toString())
-                .subscribe({
-                    _viewEvents.post(RoomProfileViewEvents.OnChangeAvatarSuccess)
-                }, {
-                    _viewEvents.post(RoomProfileViewEvents.Failure(it))
-                })
+                .subscribe(
+                        {
+                            _viewEvents.post(RoomProfileViewEvents.OnChangeAvatarSuccess)
+                        },
+                        {
+                            _viewEvents.post(RoomProfileViewEvents.Failure(it))
+                        })
```

Comment:
We may put the last parentheses to the new line? :)

Reply:
Done